_field-types/supported-field-types/match-only-text.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Match-only text
+nav_order: 61
+has_children: false
+parent: String field types
+grand_parent: Supported field types
+---
+
+# Match-only text field type
+
+A `match_only_text` field is a variant of a text field designed for full-text search when scoring and positional information of terms within a document are not critical.
+
+A `match_only_text` field is different from a `text` field in the following ways:
+
+ - Omits storing positions, frequencies, and norms, reducing storage.
+ - Disables scoring so all matching documents receive a constant score of 1.0.
+ - Supports all query types except interval and span queries.
+
+Choose the `match_only_text` field type to prioritize efficient full-text search over complex ranking and positional queries while optimizing storage costs. Using `match_only_text` creates significantly smaller indexes, which translates to lower storage costs, especially for large datasets. 
+
+Use a `match_only_text` field when you need to quickly find documents containing specific terms without the overhead of storing frequencies and positions. The `match_only_text` field type is not the best choice for ranking results based on relevance or for queries that rely on term proximity or order, like interval or span queries. While this field type does support phrase queries, their performance isn't as efficient as with the `text` field type. If identifying exact phrases or their locations within documents is essential, use the `text` field type instead.
+
+## Example
+
+Create a mapping with a `match_only_text` field:
+
+```json
+PUT movies
+{
+  "mappings" : {
+    "properties" : {
+      "title" : {
+        "type" :  "match_only_text"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Parameters
+
+While `match_only_text` supports most parameters available for text fields, modifying most of them can be counterproductive. This field type thrives on simplicity and efficiency, minimizing data stored in the index to optimize storage costs. Therefore, keeping the default settings is generally the best approach. Any adjustments beyond analyzer settings can reintroduce overhead and negate the efficiency benefits of `match_only_text`.
+
+The following table lists all parameters available for `match_text_only` fields.
+
+Parameter | Description
+:--- | :---
+`analyzer` | The analyzer to be used for this field. By default, it will be used at index time and at search time. To override it at search time, set the `search_analyzer` parameter. Default is the `standard` analyzer, which uses grammar-based tokenization and is based on the [Unicode Text Segmentation](https://unicode.org/reports/tr29/) algorithm.
```

Comment:
```suggestion
`analyzer` | The analyzer to be used for the field. By default, it will be used at index time and at search time. To override it at search time, set the `search_analyzer` parameter. Default is the `standard` analyzer, which uses grammar-based tokenization and is based on the [Unicode Text Segmentation](https://unicode.org/reports/tr29/) algorithm.
```